test/sql/constraint.sql

```diff
@@ -120,6 +120,32 @@ ALTER TABLE hyper_unique_with_looooooooooooooooooooooooooooooooooooong_name
 ADD CONSTRAINT hyper_unique_invalid UNIQUE (device_id);
 \set ON_ERROR_STOP 1
 
+----------------------- RENAME CONSTRAINT  ------------------
+
+ALTER TABLE hyper_unique_with_looooooooooooooooooooooooooooooooooooong_name
+RENAME CONSTRAINT hyper_unique_with_looooooooooooooooooooooooooooooooooo_time_key TO new_name;
+
```

Comment:
Maybe `show_constraints` here? Or why rename twice?

Reply:
I renamed twice to rename an previously-renamed constraint. Figured that's more likely to catch errors.

Reply:
I don't think we need to show the intermediate state.

Reply:
Can't hurt to have it shown as an intermediate